lib/board.rb

```diff
@@ -0,0 +1,68 @@
+require 'pry'
+class Board
+
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ")
+  end
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end
+
+
+  def display
+      puts " #{@cells [0]} | #{@cells [1]} | #{@cells [2]} "
+      puts "-----------"
+      puts " #{@cells [3]} | #{@cells [4]} | #{@cells [5]} "
+      puts "-----------"
+      puts " #{@cells [6]} | #{@cells [7]} | #{@cells [8]} "
+  end
+
+  def position(input)
+    board_position = input.to_i - 1
+    @cells[board_position]
+  end
+
+  def full?
+    @cells.none? do |check|
+      if check == " "
+        true
+      end
+    end
+  end
+
+  def turn_count
+    counter = 0
+    @cells.each do |move|
+      if move == "X" || move == "O"
+        counter += 1
+        end
+      end
+      counter
+    end
+
+    def taken?(input)
+      index = input.to_i - 1
+      if @cells[index] == " " || @cells[index] == "" || @cells[index] == nil
+        false
+      elsif @cells[index] == "X" || @cells[index] == "O"
+        true
+      end
+    end
+
+    def valid_move?(input)
+      index = input.to_i - 1
+      if taken?(input) == false && index.between?(0,8)
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression. 
Reverse (or negate) the boolean with the `!` (bang) operator.

```
 def valid_move?(input)
  index = input.to_i - 1
  !taken?(input)  && index.between?(0,8)
end
```

You may also want to consider creating an `input_to_index` method to take care of the conversion.